doc/source/getting_started/faqs.rst

```diff
@@ -117,22 +117,29 @@ all PyFluent packages in a Python *virtual environment*:
    .. code:: bash
 
    .. vale off
-   
-    python -m venv venv   # Set up a local virtual environment
-    venv\Scripts\activate   # Activate the virtual environment on Windows
-    source venv/bin/activate.csh   # Activate the virtual environment on Linux (csh)
-    . venv/bin/activate   # Activate the virtual environment on Linux (bash)
 
-   .. vale on
+      # Set up a local virtual environment
```

Comment:
Messed around with this a bit but still needed to deactivate vale.  Strange.

Reply:
@dnwillia-work I asked Roberto to look at this when it finds a few minutes. He too thought it was strange.